src/commons/utils/JsSlangHelper.ts

```diff
@@ -98,10 +98,16 @@ export function visualizeEnv({ context }: { context: Context }) {
   }
 }
 
-export function highlightLine(line: number | undefined) {
+export function highlightClean() {
   if ((window as any).Inspector) {
     (window as any).Inspector.highlightClean();
-    // if number is undefined it just clears the highlighting.
+  } else {
+    throw new Error('Inspector not loaded');
+  }
+}
+
+export function highlightLine(line: number) {
+  if ((window as any).Inspector) {
```

Comment:
Ideally, I would type the `window` object so that there is no need to cast to `any`. However, this PR itself is already far beyond the scope of my work and I have deadlines to meet.

Reply:
Created a follow-up issue #2273.